pkg/semver/semver.go

```diff
@@ -18,12 +18,16 @@ package semver
 
 import (
 	"encoding/json"
+	"flag"
 	"fmt"
 	"strconv"
 
 	semverlib "github.com/Masterminds/semver/v3"
 )
 
+// ensure that Semver implements flag.Value interface.
+var _ flag.Value = &Semver{}
```

Comment:
Consider wrapping in a function perhaps 

“ v, ok:=i.(Somether)”

Then an if statement to error based on that

Reply:
This is just to have a compile-time check that `Semver` implements `flag.Value` interface.